offchain-modules/packages/x/src/handlers/eth.ts

```diff
@@ -466,6 +681,7 @@ export class EthHandler {
     void this.watchNewBlock();
 
     this.handleUnlockRecords();
+    this.handleMintRecords();
```

Comment:
Can not concurrent execute with `this.handleUnlockRecord`

Reply:
> Can not concurrent execute with `this.handleUnlockRecord`

fix with a new task scheduler.